frontend/src/app/page.js

```diff
@@ -1,15 +1,16 @@
-import AboutSection from '@/components/home/AboutSection';
-import HeroSection from '@/components/home/HeroSection';
-import JobListings from '@/components/home/JobListings';
-import ServedCompaniesSection from '@/components/home/ServedCompaniesSection';
+
+import Pricing from "../app/prices/page";
+import AboutSection from "@/components/home/AboutSection";
+import HeroSection from "@/components/home/HeroSection";
+import ServedCompaniesSection from "@/components/home/ServedCompaniesSection";
 
 export default function Home() {
-	return (
-		<>
-			<HeroSection />
-			<AboutSection />
-			<ServedCompaniesSection />
-			<JobListings />
-		</>
-	);
+  return (
+    <>
+      <HeroSection />
+      <AboutSection />
+      <Pricing />
```

Comment:
There is no need to include the pricing component within the home page , because you have created a route for it.